.circleci/config.yml

```diff
@@ -75,10 +75,11 @@ references:
           DATABASE_URL: "postgres://postgres@localhost/hmpps-book-secure-move-api"
           EXTERNAL_URL: mocked_in_tests
           ENCRYPTOR_SALT: "*\xDB\x9E\a\x1ES1\xB7\xE3\xB8{\xF9Fl\x8Fnn?\x83\xC6\xBBR\x9Eq\x0F\x01./U\x0E\xF1\xDE"
-      - image: circleci/postgres:10.4
+      - image: circleci/postgres:10-alpine-ram
         environment:
           POSTGRES_USER: postgres
           POSTGRES_DB: hmpps-book-secure-move-api
+          LANG: C.utf8
```

Comment:
is this similar to the collation on production dbs?

Reply:
so, it turns out production is a bit of a hodge-podge of `C`, `UTF-8` and `en_US.UTF-8` collations :-(

```
Move.connection.execute("SELECT name, setting FROM pg_settings WHERE category ~ 'Locale'").each {|r| puts r}
{"name"=>"client_encoding", "setting"=>"UTF8"}
{"name"=>"DateStyle", "setting"=>"ISO, MDY"}
{"name"=>"default_text_search_config", "setting"=>"pg_catalog.simple"}
{"name"=>"extra_float_digits", "setting"=>"0"}
{"name"=>"IntervalStyle", "setting"=>"postgres"}
{"name"=>"lc_collate", "setting"=>"en_US.UTF-8"}
{"name"=>"lc_ctype", "setting"=>"en_US.UTF-8"}
{"name"=>"lc_messages", "setting"=>""}
{"name"=>"lc_monetary", "setting"=>"C"}
{"name"=>"lc_numeric", "setting"=>"C"}
{"name"=>"lc_time", "setting"=>"C"}
{"name"=>"server_encoding", "setting"=>"UTF8"}
{"name"=>"TimeZone", "setting"=>"UTC"}
{"name"=>"timezone_abbreviations", "setting"=>"Default"}
```

That being said, I think its fine to merge this now, and then tidy up the settings on production separately